s3config.json

```diff
@@ -53,6 +53,8 @@
     { "Condition": { "KeyPrefixEquals": "services/" },                                                         "Redirect": { "HostName": "$REDIR_HOSTNAME", "ReplaceKeyPrefixWith": "mesosphere/dcos/services/", "HttpRedirectCode": "301" } },
     { "Condition": { "KeyPrefixEquals": "version-policy/" },                                                   "Redirect": { "HostName": "$REDIR_HOSTNAME", "ReplaceKeyPrefixWith": "mesosphere/dcos/version-policy/", "HttpRedirectCode": "301" } },
     { "Condition": { "KeyPrefixEquals": "support" },                                                           "Redirect": { "HostName": "support.d2iq.com", "Protocol": "https", "HttpRedirectCode": "301" } },
-    { "Condition": { "KeyPrefixEquals": "dkp/kommander/1.4/Architecture/" },                                "Redirect": { "HostName": "$REDIR_HOSTNAME", "ReplaceKeyPrefixWith": "dkp/kommander/1.4/architecture/", "HttpRedirectCode": "301" } }
+    { "Condition": { "KeyPrefixEquals": "dkp/kommander/1.4/Architecture/" },                                   "Redirect": { "HostName": "$REDIR_HOSTNAME", "ReplaceKeyPrefixWith": "dkp/kommander/1.4/architecture/", "HttpRedirectCode": "301" } },
+    { "Condition": { "KeyPrefixEquals": "dkp/kommander/2.1/install/custom-domain/" },                          "Redirect": { "HostName": "$REDIR_HOSTNAME", "ReplaceKeyPrefixWith": "dkp/kommander/2.1/install/configuration/custom-domain/", "HttpRedirectCode": "301" } },
+    { "Condition": { "KeyPrefixEquals": "dkp/kommander/2.1/install/http-proxy/" },                             "Redirect": { "HostName": "$REDIR_HOSTNAME", "ReplaceKeyPrefixWith": "dkp/kommander/2.1/install/configuration/http-proxy/", "HttpRedirectCode": "301" } }
```

Comment:
So unfortunately, we're limited by AWS for the number of redirects we can have - so, we can't have these because we're already at our limit. :(

I don't know if we've ever had these directly linked any where ... so, I don't know if there is a big risk for this getting moved around without redirecting.

Or we could have just sorta blah pages for custom-domain  and http proxy on the install section to just say "click here for the real thing"

Reply:
Ah yeah that’s right. In the past I’ve gotten around this by using Lambda@Edge for redirects instead of the s3 website redirects. Do you think it’s worth filing a ticket?  

In the mean time, I’ll get rid of these changes.